src/test/java/com/gamedoora/backend/studioservices/repository/StudioRepositoryTest.java

```diff
@@ -0,0 +1,32 @@
+package com.gamedoora.backend.studioservices.repository;
+import com.gamedoora.model.dao.*;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+@DataJpaTest(properties = {"spring.cloud.config.enabled=false"})
+public class StudioRepositoryTest {
+
+    @Autowired
+    public StudioRepository studioRepository;
+
+    @Test
+    public void testFindByRegistration() {
```

Comment:
test looks good, just one addition, since it is a list, the recommended way is adding more than one object into the repository and then fetching them into the list and verify the size and properties of those studio object using assertEquals.